tests/integration/test_pr_agent_workflow_specific.py

```
@@ -0,0 +1,492 @@
+"""
+Comprehensive tests specifically for pr-agent.yml workflow.
+Tests the duplicate key fix and PR Agent-specific functionality.
+"""
+
+import pytest
+import yaml
+import re
+from pathlib import Path
+from typing import Dict, Any
```

Ruff (0.14.5)

`typing.Dict` is deprecated, use `dict` instead

(UP035)